tooling/nargo_cli/tests/execution_success/brillig_slices/src/main.nr

```diff
@@ -113,15 +113,7 @@ unconstrained fn merge_slices_else(x: Field) {
 // Test returning a merged slice without a mutation
 unconstrained fn merge_slices_return(x: Field, y: Field) -> [Field] {
     let slice = [0; 2];
-    if x != y {
-        if x != 20 {
-            slice.push_back(y)
-        } else {
-            slice
-        }
-    } else {
-        slice
-    }
+    if x != y { if x != 20 { slice.push_back(y) } else { slice } } else { slice }
```

Comment:
This is rather aggressive on putting ifs used as expressions on one line. I wonder what other formatters do here.